shorthand.module

```diff
@@ -110,18 +110,27 @@ function shorthand_node_update($node) {
 function update_node_data($node) {
   if (isset($node->shorthand_story_id)) {
     $node->original = isset($node->original) ? $node->original : NULL;
-    $shorthand_id = $node->shorthand_story_id[$node->language][0]['value'];
-    $node_id = $node->uid;
-    $story = sh_copy_story($node_id, $shorthand_id);
-    $head = _shorthand_fix_content_paths($story['url'], file_get_contents($story['path'] . '/component_head.html'));
-    $node->shorthand_story_head[$node->language][0]['value'] = $head;
-    $body = _shorthand_fix_content_paths($story['url'], file_get_contents($story['path'] . '/component_article.html'));
-    $node->shorthand_story_body[$node->language][0]['value'] = $body;
+
+    $story = sh_copy_story($node->nid, $node->shorthand_story_id[$node->language][0]['value']);
```

Comment:
@simultech I've changed UID to NID here because if the comments in `sh_copy_story()` https://github.com/dakala/shorthand_connect_drupal/blob/3e18ea00968a36dbf351b6214ecb63ece69c75c3/includes/api.php#L83-L84 The sub-directory should be the ID of the node rather than the author.

I've cross-checked the Wordpress code and it's using the POST_ID here.